lib/node/api.js

```diff
@@ -781,18 +781,19 @@ const getTransactionDetails = async (txids, getConfirmations = false) => {
   const txDetails = [];
   const invalidTxids = [];
   const confirmations = [];
-  const promises = txids.map(async (txid) => {
+  const txPromises = txids.map(async (txid) => {
```

Comment:
We can remove the async here.

Reply:
Fixed here: https://github.com/imthe-1/multi-dimension-identity-protocol/pull/4/files/c2c09fb7cdf6ce9928e8381e61347c6d30d69f3c..f96fa1ac8fedb5403059bd5121c9a430c3ae36c2